### ISSUE TITLE
Homebase state is set to triggered for a second before going to the assigned status 

### ISSUE THREAD

**Issue by mmb25** [fuatakgun/eufy_security#581]
Please check Services, Known Bug / Issues and Troubleshooting over here first: https://github.com/fuatakgun/eufy_security/blob/master/README.md

**Describe the bug**
I did have this integration working for a long time now, 
Today, after replacing my homebase 2 with a homebase 3. This is what works differently.

When I change the Alarm Mode from the eufy app (for whatever mode to whatever mode) , The homebase 3 entity is set to Triggered before going to the target state.

(So lets say that I have my eufy on HOME mode, and I set it to AWAY) The entity will show like this:

1)HOME
2) TRIGGERED
3) AWAY

**To Reproduce**
Just change from one status to another when using homebase 3
**Expected behavior**

**Screenshots**

**Hardware Information about Home Assistant

**Hardware Information about Cameras
- Camera Model
- Live Streaming Protocol: RTSP or P2p

**Additional context**

**Comment by mmb25** [comment]
Also, any workaround would be really appreciated

**Comment by bjgilberd** [comment]
Same issue here.  Although I am on Homebase 2.
[Eufy HA integration.pdf](https://github.com/fuatakgun/eufy_security/files/10281695/Eufy.HA.integration.pdf)

**Comment by fuatakgun** [comment]
My local setup is not working but i had just released a new version, please give it a try.

**Comment by bjgilberd** [comment]
![Eufy HA integration](https://user-images.githubusercontent.com/80750993/209013978-049e6b63-29b0-44ea-ab90-f51b38f528ff.png)
I updated (2.9.1), restarted and tested but the results seem to be the same.   Not sure if these logs from the Add-In are useful:

lookup: 1671661097700,
    connect: 1671661097899,
    secureConnect: 1671661098097,
    upload: 1671661098098,
    response: 1671661098297,
    end: 1671661098298,
    error: undefined,
    abort: undefined,
    phases: {
      wait: 0,
      dns: 1,
      tcp: 199,
      tls: 198,
      request: 1,
      firstByte: 199,
      download: 1,
      total: 599
    }
  }
}
error stack:
• index.js:118 <anonymous>
    node_modules/got/dist/source/as-promise/index.js:118:42
• task_queues:96 processTicksAndRejections
    node:internal/process/task_queues:96:5
2022-12-21 22:18:18.301  INFO  No devices found. 
2022-12-21 22:18:18.329  ERROR EufyError - Code: undefined Message: undefined (HTTP Code: 403) - requestUrl: https://security-app.eufylife.com/v1/house/device_list 
2022-12-21 22:18:18.330  ERROR Devices - Generic Error: 
 EufyError  Code: undefined Message: undefined (HTTP Code: 403)
details:
{
  code: 'ERR_NON_2XX_3XX_RESPONSE',
  timings: {
    start: 1671661097716,
    socket: 1671661097716,
    lookup: 1671661097717,
    connect: 1671661097921,
    secureConnect: 1671661098124,
    upload: 1671661098124,
    response: 1671661098327,
    end: 1671661098328,
    error: undefined,
    abort: undefined,
    phases: {
      wait: 0,
      dns: 1,
      tcp: 204,
      tls: 203,
      request: 0,
      firstByte: 203,
      download: 1,
      total: 612
    }
  }
}
error stack:
• index.js:118 <anonymous>
    node_modules/got/dist/source/as-promise/index.js:118:42
• task_queues:96 processTicksAndRejections
2022-12-21 22:18:18.331  INFO  No devices found. 
    node:internal/process/task_queues:96:5
2022-12-21 22:20:29.066  INFO  Alarm mode for station T8010P1320310A4D changed to: AWAY 
2022-12-21 22:20:29.150  ERROR Invalid Property wifiSignalLevel error 
 InvalidPropertyError  Property wifiSignalLevel invalid
error stack:
• device.ts:507 getPropertyMetadata
    node_modules/eufy-security-client/src/http/device.ts:507:15
• device.ts:76 updateProperty
    node_modules/eufy-security-client/src/http/device.ts:76:48
• device.ts:120 handlePropertyChange
    node_modules/eufy-security-client/src/http/device.ts:120:18
• device.ts:2106 handlePropertyChange
    node_modules/eufy-security-client/src/http/device.ts:2106:15
• device.ts:76 updateProperty
    node_modules/eufy-security-client/src/http/device.ts:76:22
• device.ts:138 updateRawProperty
    node_modules/eufy-security-client/src/http/device.ts:138:30
• device.ts:92 <anonymous>
    node_modules/eufy-security-client/src/http/device.ts:92:18
• device.ts:90 updateRawProperties
    node_modules/eufy-security-client/src/http/device.ts:90:29
• eufysecurity.ts:698 <anonymous>
    node_modules/eufy-security-client/src/eufysecurity.ts:698:20
2022-12-21 22:22:14.567  INFO  Alarm mode for station T8010P1320310A4D changed to: HOME

**Comment by mmb25** [comment]
Hello, same here:

Sorry for the language, its in Spanish, but "Disparada" means triggered
<img width="923" alt="image" src="https://user-images.githubusercontent.com/93614707/209187053-d543cbfa-1962-45ce-ab16-e32af70110a1.png">

I am a S.Eng also myself, if you want a zoom call at some point to troubleshoot it I am open to help.

Also, if you want help contributing to this project, also happy to help, really appreciate your work here, Eufy its actually the core of most of my automations.

For the rest of the people, I do have a workaround to avoid false alarms when changing the status.


1) If I want to manually update the status, I have set a button in HA, that essentially disables all the alarm related automations, to then, set the homebase to DISSARMED/ AWAY , then wait 5 sec, and then enable the automations.
2) I also have noticed that this doesnt get "triggered" when the change of mode its done as part of the eufy scheduled.

I dont' fully understand why this is failing, as it wasn't with eufy homebase 2 in my case, and it wasn't updated your integration, I suspect that something has been changed in the eufy firmware/eufy app itself.
